src/packages/cli/src/__tests__/__helpers__/integrationTest.ts

```diff
@@ -122,71 +146,44 @@ type Settings = {
    * @dynamicDefault The result of `@prisma/fetch-engine#getLatestTag`
    */
   engineVersion?: MaybePromise<string>
+  /**
+   * After a test scenario is done, should its temporary directory be removed from disk?
+   */
+  cleanupTempDirs?: boolean
 }
 
 /**
  * Integration test keyword arguments
  */
-type Input<Client> = {
+export type Input<Client = any> = {
   database: Database<Client>
   scenarios: Scenario[]
   settings?: Settings
 }
 
-process.env.SKIP_GENERATE = 'true'
-const pkgDir = pkgup.sync() || __dirname
-const engine = new IntrospectionEngine()
+type ScenarioState<Client = any> = {
+  scenario: Scenario
+  ctx: Context
+  database: Input<Client>['database']
+  db: Client
+  prisma: any
+  input: Input<Client>
+}
 
-export function integrationTest<Client>(input: Input<Client>) {
-  type ScenarioState = {
-    scenario: Scenario
-    ctx: Context
-    database: Input<Client>['database']
-    db: Client
-    prisma: any
-  }
+/**
+ * Run introspection integration test
+ */
+export function introspectionIntegrationTest<Client>(input: Input<Client>) {
+  const kind = 'introspection'
```

Comment:
New label tier used throughout to separate introspection from runtime tests, since both share the same base of scenarios.